### PR TITLE
Added program arguments to .run folder

### PR DESCRIPTION
**Pull request by KristianTrifork**

### CODE REVIEW

File: EnrichStream/.run/EnrichStream.run.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            <env name="KAFKA_CLIENT_SECRET" value="default-access-secret" />
+            <env name="KAFKA_SCOPE" value="kafka" />
+            <env name="SCHEMA_REGISTRY_CLIENT_ID" value="default-access" />
+            <env name="SCHEMA_REGISTRY_CLIENT_SECRET" value="default-access-secret" />
+            <env name="SCHEMA_REGISTRY_SCOPE" value="schema-registry" />
+        </envs>
+        <option name="INCLUDE_PROVIDED_SCOPE" value="true" />
+        <option name="MAIN_CLASS_NAME" value="cheetah.example.job.EnrichStreamJob" />
+        <module name="${lowerCaseJobName}" />
+        <option name="PROGRAM_PARAMETERS" value="--kafka-bootstrap-servers localhost:9092 --kafka-group-id EnrichStreamJson-group-id --schema-registry-url http://localhost:8081/apis/ccompat/v7 --input-kafka-topic EnrichStreamEnrichTopic --input-kafka-topic EnrichStreamInputTopic --output-kafka-topic EnrichStreamOutputTopic" />
```
Comment (rnotrifork) [inline EnrichStream/.run/EnrichStream.run.xml:16]: Should the *-kafka-topic parameters match the parameters declared in the docker-compose? (Should postfix be included?)
In the docker compose file the input kafka topics are named input-kafka-topic-**enriching** and input-kafka-topic-**main-source**.

File: ExternalLookup/.run/ExternalLookup.run.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            <env name="KAFKA_CLIENT_SECRET" value="default-access-secret" />
+            <env name="KAFKA_SCOPE" value="kafka" />
+            <env name="SCHEMA_REGISTRY_CLIENT_ID" value="default-access" />
+            <env name="SCHEMA_REGISTRY_CLIENT_SECRET" value="default-access-secret" />
+            <env name="SCHEMA_REGISTRY_SCOPE" value="schema-registry" />
+        </envs>
+        <option name="INCLUDE_PROVIDED_SCOPE" value="true" />
+        <option name="MAIN_CLASS_NAME" value="cheetah.example.job.ExternalLookupJob" />
+        <module name="${lowerCaseJobName}" />
+        <option name="PROGRAM_PARAMETERS" value="--kafka-bootstrap-servers localhost:9092 --kafka-group-id ExternalLookup-group-id --schema-registry-url http://localhost:8081/apis/ccompat/v7 --input-kafka-topic-main-source ExternalLookupInputTopic --output-kafka-topic EnrichStreamOutputTopic" />
```
Comment (rnotrifork) [inline ExternalLookup/.run/ExternalLookup.run.xml:16]: Possibly missing postfix for output-kafka-topic.

File: JsonToAvro/.run/JsonToAvro.run.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            <env name="KAFKA_CLIENT_SECRET" value="default-access-secret" />
+            <env name="KAFKA_SCOPE" value="kafka" />
+            <env name="SCHEMA_REGISTRY_CLIENT_ID" value="default-access" />
+            <env name="SCHEMA_REGISTRY_CLIENT_SECRET" value="default-access-secret" />
+            <env name="SCHEMA_REGISTRY_SCOPE" value="schema-registry" />
+        </envs>
+        <option name="INCLUDE_PROVIDED_SCOPE" value="true" />
+        <option name="MAIN_CLASS_NAME" value="cheetah.example.job.JsonToAvroJob" />
+        <module name="${lowerCaseJobName}" />
+        <option name="PROGRAM_PARAMETERS" value="--kafka-bootstrap-servers localhost:9092 --kafka-group-id jsonToAvro-group-id --schema-registry-url http://localhost:8081/apis/ccompat/v7 --input-kafka-topic-main-source jsonToAvroInputTopic --output-kafka-topic-value jsonToAvroOutputTopic-value" />
```
Comment (rnotrifork) [inline JsonToAvro/.run/JsonToAvro.run.xml:16]: Possibly wrong postfix for output-kafka-topic.

File: TransformAndStore/.run/TransformAndStore.run.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+            <env name="KAFKA_CLIENT_SECRET" value="default-access-secret" />
+            <env name="KAFKA_SCOPE" value="kafka" />
+            <env name="SCHEMA_REGISTRY_CLIENT_ID" value="default-access" />
+            <env name="SCHEMA_REGISTRY_CLIENT_SECRET" value="default-access-secret" />
+            <env name="SCHEMA_REGISTRY_SCOPE" value="schema-registry" />
+        </envs>
+        <option name="INCLUDE_PROVIDED_SCOPE" value="true" />
+        <option name="MAIN_CLASS_NAME" value="cheetah.example.job.TransformAndStoreJob" />
+        <module name="${lowerCaseJobName}" />
+        <option name="PROGRAM_PARAMETERS" value="--kafka-bootstrap-servers localhost:9092 --kafka-group-id transform-and-store-group-id --schema-registry-url http://localhost:8081/apis/ccompat/v7 --input-kafka-topic-main-source TransformAndStoreInputTopic" />
```
Comment (rnotrifork) [inline TransformAndStore/.run/TransformAndStore.run.xml:16]: Possibly missing opensearch parameters.

File: EnrichStream/.run/EnrichStream.run.xml (hunk 1 above)
Comment (KristianTrifork) [reply EnrichStream/.run/EnrichStream.run.xml:16]: I am fixing that on my branch

File: ExternalLookup/.run/ExternalLookup.run.xml (hunk 2 above)
Comment (KristianTrifork) [reply ExternalLookup/.run/ExternalLookup.run.xml:16]: Also fixed on my branch

**Review by rnotrifork — APPROVED** [review]

**Pull request merged**